README.md

```diff
@@ -57,11 +62,15 @@ eclipse
 
 ![step4](./images/4-ImportFromGit.png)
 
-#### Step 4.5: Tell Git which repo to use (core-v-mcu-eclipse-cli-test)
+#### Step 4.5: Specify the Source Git Repository
+Provide the full URL path to `core-v-mcu-cli-test`.
```

Comment:
we should provide the full link which can be copied https://github.com/openhwgroup/core-v-mcu-cli-test

also, the user should just be 'git' with no password (not full user credentials as shown in the image)